### ISSUE TITLE
fix tile getter

### ISSUE THREAD

**Issue by mdsumner** [hypertidy/ceramic#49]
use regex to get the z/y/x components

**Comment by mdsumner** [comment]
```
ceramic_tiles(zoom = 14)
# A tibble: 55 × 11
   tile_x tile_y  zoom type             version source         fullname                                         xmin     xmax      ymin      ymax
    <int>  <int> <int> <chr>            <chr>   <chr>          <fs::path>                                      <dbl>    <dbl>     <dbl>     <dbl>
 1  11738  12531    14 mapbox.satellite v4      api.mapbox.com ….com/v4/mapbox.satellite/14/11738/12531.jpg 8.6735e6 8.6759e6 -1.0616e7 -1.0613e7
 2  11738  12532    14 mapbox.satellite v4      api.mapbox.com ….com/v4/mapbox.satellite/14/11738/12532.jpg 8.6735e6 8.6759e6 -1.0618e7 -1.0616e7
 3  11739  12531    14 mapbox.satellite v4      api.mapbox.com ….com/v4/mapbox.satellite/14/11739/12531.jpg 8.6759e6 8.6784e6 -1.0616e7 -1.0613e7
 4  11739  12532    14 mapbox.satellite v4      api.mapbox.com ….com/v4/mapbox.satellite/14/11739/12532.jpg 8.6759e6 8.6784e6 -1.0618e7 -1.0616e7
 5  11740  12531    14 mapbox.satellite v4      api.mapbox.com ….com/v4/mapbox.satellite/14/11740/12531.jpg 8.6784e6 8.6808e6 -1.0616e7 -1.0613e7
 6  11740  12532    14 mapbox.satellite v4      api.mapbox.com ….com/v4/mapbox.satellite/14/11740/12532.jpg 8.6784e6 8.6808e6 -1.0618e7 -1.0616e7
 7  11741  12531    14 mapbox.satellite v4      api.mapbox.com ….com/v4/mapbox.satellite/14/11741/12531.jpg 8.6808e6 8.6832e6 -1.0616e7 -1.0613e7
 8  11741  12532    14 mapbox.satellite v4      api.mapbox.com ….com/v4/mapbox.satellite/14/11741/12532.jpg 8.6808e6 8.6832e6 -1.0618e7 -1.0616e7
 9  11742  12531    14 mapbox.satellite v4      api.mapbox.com ….com/v4/mapbox.satellite/14/11742/12531.jpg 8.6832e6 8.6857e6 -1.0616e7 -1.0613e7
10  14880  10299    14 mapbox.satellite v4      api.mapbox.com ….com/v4/mapbox.satellite/14/14880/10299.jpg 1.6359e7 1.6361e7 -5.1561e6 -5.1537e6
# … with 45 more rows
# ℹ Use `print(n = ...)` to see more rows
Warning messages:
1: In eval_tidy(xs[[j]], mask) : NAs introduced by coercion
2: In eval_tidy(xs[[j]], mask) :
  NAs introduced by coercion to integer range
3: In eval_tidy(xs[[j]], mask) : NAs introduced by coercion
```

**Comment by mdsumner** [comment]
fixed with str_extract b8292d8e0d2d8244ea7e6a6126e380961fa93416
